javalib/src/main/scala/java/security/MessageDigest.scala

```diff
@@ -0,0 +1,32 @@
+package java.security
+
+// This is only a partial, minimal implementation.
+// TODO: Port me from Apache Harmony
```

Comment:
I added a note to the issue tracking progress on the zip/jar front (#932) to remind us to eventually port those for real.